### DISCUSSION TITLE
Allow setting Indexes

### DISCUSSION THREAD

**Discussion by WoLfulus** [directus/directus#3024] [Feature Requests]
I'm trying to setup a really data heavy collection that needs several indexes in order to make them performant for the reports I need to make. I haven't found a way to setup indexes for those columns.

Compound indexes is a must in this case.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
